presto-base-jdbc/src/main/java/com/facebook/presto/plugin/jdbc/JdbcMetadata.java

```diff
@@ -66,6 +72,21 @@ public JdbcTableHandle getTableHandle(ConnectorSession session, SchemaTableName
     }
 
     @Override
+    public List<ConnectorTableLayoutResult> getTableLayouts(ConnectorSession session, ConnectorTableHandle table, Constraint<ColumnHandle> constraint, Optional<Set<ColumnHandle>> desiredColumns)
+    {
+        JdbcTableHandle tableHandle = checkType(table, JdbcTableHandle.class, "table");
+        ConnectorTableLayout layout = new ConnectorTableLayout(new JdbcTableLayoutHandle(tableHandle, constraint.getSummary()));
+        return ImmutableList.of(new ConnectorTableLayoutResult(layout, constraint.getSummary()));
+    }
+
+    @Override
+    public ConnectorTableLayout getTableLayout(ConnectorSession session, ConnectorTableLayoutHandle handle)
+    {
+        JdbcTableLayoutHandle layout = checkType(handle, JdbcTableLayoutHandle.class, "layout");
+        return new ConnectorTableLayout(layout);
```

Comment:
This can just pass `handle` to `ConnectorTableLayout`, no need to check the type.